test/OpenTelemetry.Exporter.Geneva.Tests/GenevaTraceExporterTests.cs

```diff
@@ -81,6 +81,32 @@ public void GenevaTraceExporter_constructor_Invalid_Input()
                     },
                 });
             });
+
+            // Supported types for PrepopulatedFields should not throw an exception
+            var exception = Record.Exception(() =>
+            {
+                new GenevaExporterOptions
```

Comment:
This is not using an exporter ctor anymore. The tests need refactoring. We need to add a dedicated test class for `GenevaExporterOptions`. I'll not cover them in this PR.